source/assets/css/basic.less

```diff
@@ -1195,6 +1195,8 @@ input#search-query {
 
 .devdocs {
     font-size: 15px;
+    min-height: 800px;
+    min-height: 80vh;
```

Comment:
should have googled that. stupid hipster css alien technologies nowadays *grumpy*